library/src/main/java/io/appwrite/Client.kt

```diff
@@ -69,7 +70,7 @@ class Client @JvmOverloads constructor(
             "content-type" to "application/json",
             "origin" to "appwrite-android://${context.packageName}",
             "user-agent" to "${context.packageName}/${appVersion}, ${System.getProperty("http.agent")}",
-            "x-sdk-version" to "appwrite:android:0.0.1",            
+            "x-sdk-version" to "appwrite:android:${BuildConfig.SDK_VERSION}",            
             "x-appwrite-response-format" to "0.8.0"
```

Comment:
Not sure but this might need to be changed to 0.9.0 based on the readme updates

```suggestion
            "x-appwrite-response-format" to "0.9.0"
```

Reply:
Updated this! Thanks @abnegate :smile: